### PR TITLE
Send back a 409 for duplicate UUIDs

### PR DESCRIPTION
**Pull request by wssheldon**

### CODE REVIEW

File: src/dispatch/signal/views.py [hunk 1]
```diff
@@ -104,12 +105,24 @@ def create_signal_instance(
...
-    signal_instance.signal = signal
-    db_session.commit()
-    return signal_instance
+    try:
+        signal_instance = signal_service.create_instance(
+            db_session=db_session, signal_instance_in=signal_instance_in
+        )
+    except IntegrityError as e:
+        if type(e.orig) is UniqueViolation:
+            msg = f"The id: {signal_instance_in.id} already exists in the database. Please provide a unique id."
```
Comment (mvilanova) [inline src/dispatch/signal/views.py:114]:
```suggestion
            msg = f"Signal instance id {signal_instance_in.id} already exists in the database. Please provide a unique id."
```

Comment (mvilanova) [reply src/dispatch/signal/views.py:114]: Will the info in this message be enough to troubleshoot on the other side?

**Review by mvilanova — APPROVED** [review]

File: src/dispatch/signal/views.py (hunk 1 above)
Comment (wssheldon) [reply src/dispatch/signal/views.py]: Probably, it could be improved though. I'll add some additional information.

**Comment by wssheldon** [comment]
Abandoning this PR in favor of #3564

**Pull request closed without merging**